pkg/move/cmd.go

```diff
@@ -6,17 +6,23 @@ import (
 )
 
 const (
-	sourceFolderFlag = "source"
-	targetFolderFlag = "target"
-	workFolderFlag   = "work"
-	technologyFlag   = "technology"
+	sourceFolderFlag       = "source"
+	targetFolderFlag       = "target"
+	workFolderFlag         = "work"
+	technologyFlag         = "technology"
+	configDirFlag          = "config-directory"
+	attributeFlag          = "attribute"
+	attributeContainerFlag = "attribute-container"
```

Comment:
we should introduce a new package called `configure`, we can't have everything in 1 package